code/modules/projectiles/updated_projectiles/guns/flamer.dm

```diff
@@ -550,41 +555,41 @@
 	for(var/i in loc)
 		if(++j >= 11)
 			break
-		if(isliving(i))
-			var/mob/living/I = i
-			if(istype(I,/mob/living/carbon/human))
-				var/mob/living/carbon/human/M = I
-				if(istype(M.wear_suit, /obj/item/clothing/suit/fire) || istype(M.wear_suit,/obj/item/clothing/suit/space/rig/atmos) || (istype(M.wear_suit, /obj/item/clothing/suit/storage/marine/M35) && istype(M.head, /obj/item/clothing/head/helmet/marine/pyro)))
-					M.show_message(text("Your suit protects you from the flames."),1)
-					M.adjustFireLoss(rand(0 ,burnlevel*0.25)) //Does small burn damage to a person wearing one of the suits.
-					continue
-			if(istype(I,/mob/living/carbon/Xenomorph/Queen))
-				var/mob/living/carbon/Xenomorph/Queen/X = I
-				X.show_message(text("Your extra-thick exoskeleton protects you from the flames."),1)
-				continue
-			if(istype(I,/mob/living/carbon/Xenomorph/Ravager))
-				if(!I.stat)
-					var/mob/living/carbon/Xenomorph/Ravager/X = I
-					X.plasma_stored = X.xeno_caste.plasma_max
-					X.usedcharge = 0 //Reset charge cooldown
-					X.show_message(text("<span class='danger'>The heat of the fire roars in your veins! KILL! CHARGE! DESTROY!</span>"),1)
-					if(rand(1,100) < 70)
-						X.emote("roar")
-				continue
-
+		var/atom/A = i
```

Comment:
Why not just do a `for(var/atom/A in loc)`?

Reply:
its faster